docs/content/docs/dev/table/functions/udfs.md

```diff
@@ -1778,9 +1778,9 @@ def emitUpdateWithRetract(accumulator: ACC, out: RetractableCollector[T]): Unit
 The following example shows how to use the `emitUpdateWithRetract(...)` method to emit only incremental
 updates. In order to do so, the accumulator keeps both the old and new top 2 values.
 
-If the N of Top N is big, it might be inefficient to keep both the old and new values. One way to
-solve this case is to store only the input record in the accumulator in `accumulate` method and then perform
-a calculation in `emitUpdateWithRetract`.
+{{< hint info >}}
+Note: Do not update accumulator within `emitUpdateWithRetract` because `GroupTableAggFunction` will not re-invoke `function#getAccumulators` after `function#emitValue` is invoked.
```

Comment:
> because `GroupTableAggFunction` will not re-invoke `function#getAccumulators after `function#emitValue` is invoked`

What about `because after function#emitUpdateWithRetract is invoked, GroupTableAggFunction will not re-invoke function#getAccumulators to update the latest accumulator to state `